doc/JSON_INFO.md

```diff
@@ -543,7 +543,7 @@ Mods can modify this via `add:traits` and `remove:traits`.
     [ "textbook_gaswarfare", 8, "" ] // If the name is empty, the recipe is hidden, it will not be shown in the description of the book.
 ],
 "difficulty": 3,             // Difficulty of success check
-"time": 5000,                // Time to perform recipe (where 1000 ~= 10 turns ~= 1 minute game time)
+"time": 5000,                // Time to perform recipe. An integer value will be interpreted as moves or a time string can be used.
```

Comment:
I would keep the bit explaining how 1000 = 10 turns = 1 minute, and add in what time strings are accepted.

Reply:
I think the json documentation isn't the place to explain how moves map to turns, especially when that relationship is going to change soon and integer time values are becoming legacy.

Reply:
The moves to time definition should probably still be in documentation; while it's not as good a definition in general, it is still accepted, and legacy recipes (eg in mods) will still be using it. It could go alongside your strings definition if you prefer.